proposals/000-comm-grants.md

```diff
@@ -0,0 +1,223 @@
+will a communicate to the committee how much funding is available for grants,
+based on the program's budget, which is at the discretion of the Foundation.
+
+Each proposal will be considered by members of the committee, using their
+informed judgement to determine whether accepting the submission is in the best
+interest of the Haskell community.  Among other factors, committee members
+should consider:
+1. Whether the proposal might be controversial.  It should be rejected it if
+   there is any likelihood that it will.
+2. How much difference the grant is likely to make on the probability for
+   success.  A good proposal will justify the amount asked for, which could be
+   used for paying expenses directly related to the project, removing obstacles
+   to contribution by, for example, paying for child care, or covering housing
+   and basic expenses if the alternative is to spend less time on project work.
+3. The overall impact if the effort succeeds.  In determining the impact, the
+   committee should keep in mind the long-term benefit of building expertise, if
+   the submission is likely to lead to opportunities for continuing work.
+
+Community Grant Committee members will use approval voting to determine which
+submissions to accept.   A submission is accepted if it receives majority
+support from the committee.  When a submission is not accepted, the committee
+may choose, by majority vote again, to provide feedback to the submitter on how
+their proposal could be improved.  Otherwise, the results of the vote are not
+announced, to avoid embarrassment or hard feelings.
+
+A random lottery system will then be used to award grants to accepted proposals,
+continuing until either the approved submissions are exhausted, or a submission
+is drawn which cannot be funded with the remaining budget.  The committee will
+then recommend to the executive team to award the grants selected in the
+lottery.  The executive team will have final authority to award grants.
+
+## People
+
+- The Haskell Foundation executive team and board will determine how many
+  projects are chosen by setting a budget as funds are available.
+- There will be a Community Grant Committee of five people responsible for
+  evaluating whether to accept each submission.  This will be chosen by the
+  Haskell Foundation executive team.
+- Proposals will be written and submitted by various members of the community
+  who feel their contributions would benefit from funding.
+
+## Resources
+
+The project can consume whatever resources are allocated to it, at the
+discretion of the Haskell Foundation board or executive team.
+
+## Timeline
+
+- Choosing a Community Grant Committee may require apprroximately a month.
+- Once there is a Community Grant Committee, the first round of grants can occur
+  as soon as the budget is available.  The committee will choose new grants
+  monthly after that.
+
+## Lifecycle:
+
+N/A
```

Comment:
I think there is a bit that could be said here: it would take some time (a month?) to actually organize, and then the grants program could begin, without a specified ending date.